src/index.js

```diff
@@ -72,10 +72,26 @@ class RovingTabindex {
         this._onInitListener = onModelInit.bind(this);
         this._onResetListener = onModelReset.bind(this);
 
-        el.addEventListener('navigationModelMutation', this._onMutationListener);
-        el.addEventListener('navigationModelChange', this._onChangeListener);
-        el.addEventListener('navigationModelInit', this._onInitListener);
-        el.addEventListener('navigationModelReset', this._onResetListener);
+        this._el.addEventListener('navigationModelMutation', this._onMutationListener);
+        this._el.addEventListener('navigationModelChange', this._onChangeListener);
+        this._el.addEventListener('navigationModelInit', this._onInitListener);
+        this._el.addEventListener('navigationModelReset', this._onResetListener);
+    }
+
+    set _el(el) {
+        return el;
+    }
+
+    get _el() {
```

Comment:
I'm not a fan of adding console statements in production code. If we feel the API is bad or broken or not intuitive, I would rather fix it at that level rather than put in console hints/clues. With that in mind, adding getter and setter doesn't seem to have any benefit over just reading/writing `instance._el` directly.  Perhaps an event would be better if the element gets removed?

Reply:
Or an `isDirty()` method (or similar)?